### ISSUE TITLE
SlevomatCodingStandard.Classes.ConstantSpacing removes use of Trait

### ISSUE THREAD

**Issue by holtkamp** [slevomat/coding-standard#1050]
When using `dev-master` @ PHP 7.4.7 and the following configuration:

```XML
    <rule ref="SlevomatCodingStandard.Classes.ConstantSpacing">
        <properties>
            <property name="minLinesCountBeforeWithComment" value="1"/>
        </properties>
    </rule>
```

And a class which has `use SomeTraitName;` between two properties:

```PHP
namespace Dummy;

class TestClass
{
    public const CONSTANT_1 = 'foo';

    use TestTrait;

    private const CONSTANT_2 = 'bar';

    public function __construct()
    {
        echo self::CONSTANT_1 . self::CONSTANT_2 . $this->someMethodFromTrait();
    }
}
```

The result of `phpcs` is:

```
FOUND 1 ERROR AFFECTING 1 LINE
---------------------------------------------------------------------------
 9 | ERROR | [x] Expected 0 to 1 blank lines after constant, found 3.
```

And after running `phpcbf` the file is changed to:

```PHP
namespace Dummy;

class TestClass
{
    public const CONSTANT_1 = 'foo';

    private const CONSTANT_2 = 'bar';

    public function __construct()
    {
        echo self::CONSTANT_1 . self::CONSTANT_2 . $this->someMethodFromTrait();
    }
}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/9a5e1fcb9b75edf18f50b7f777033be45ff4f948
